xlators/mgmt/glusterd/src/glusterd.c

```diff
@@ -137,14 +137,12 @@ glusterd_opinfo_init()
     return ret;
 }
 
-int
-glusterd_uuid_init()
+static int
+glusterd_uuid_init(xlator_t *this)
```

Comment:
nit: you could just passed 'conf' variable to this function instead of 'this' - and I'm not sure the GF_ASSERT() is needed?

Reply:
We still need `this` for `gf_msg()` at least, and using`THIS` introduces an extra TLS access.

Reply:
Ah, right - didn't notice the log was on INFO level.